CAIPs/caip-308.md

```diff
@@ -0,0 +1,78 @@
+---
+caip: 308
```

Comment:
```suggestion
caip: 311
```

This is the one that needed to be renamed, so sorry

Reply:
https://github.com/ChainAgnostic/CAIPs/pull/285/commits/9941270baeb8e1e2114c2692cee7e160b7db28d2